Draft blog post for my retirement from TWiR. (#542)
rust-lang/blog.rust-lang.org: posts/inside-rust/2020-03-12-twir-new-lead.md

---
layout: post

> title: "Vikrant (nasa42) is retiring from This Week in Rust. TWiR is looking for a new maintainer."

Reviewer: ```suggestion
title: "This Week in Rust is looking for a new maintainer."
```

> author: <TBD>

Reviewer: I don't mind being the author for the sake for the post, but if someone else is more appropriate, they can take it.
```suggestion
author: Erin Power
```

team: the community team <https://www.rust-lang.org/governance/teams/community>
---

> Vikrant is retiring from This Week in Rust. He joined This Week in Rust in June 2015 with issue 84 and has been part of Rust Community team since February 2018.

Reviewer: ```suggestion
Vikrant Chaudhary is retiring from [This Week in Rust][twir]. He joined This Week in Rust in June 2015 with issue 84 and has been part of Rust Community team since February 2018. We'd like to thank Vikrant for his stewardship of TWiR these past five years, and making TWiR one of the community's favourite newsletters. We wish him all the best in his future projects.

[twir]: https://this-week-in-rust.org
```

> TWiR is now looking for a new lead maintainer. To help smooth the transition, Vikrant will be writing TWiR for 4 more weeks while we onboard a new maintainer. If you're interested please, express your interest [here](https://github.com/emberian/this-week-in-rust/issues/1167).

Reviewer: ```suggestion
TWiR is now looking for a new lead maintainer. To help smooth the transition, Vikrant will be writing TWiR for 4 more weeks while we onboard a new maintainer. If you're interested please, express your interest on our [GitHub Issue](https://github.com/emberian/this-week-in-rust/issues/1167).
```

---
layout: post
title: "This Week in Rust is looking for a new maintainer."
author: Erin Power
team: the community team <https://www.rust-lang.org/governance/teams/community>
---

> Vikrant Chaudhary is retiring from [This Week in Rust][twir]. He joined This Week in Rust in June 2015 with issue 84 and has been part of Rust Community team since February 2018. We'd like to thank Vikrant for his stewardship of TWiR these past five years, and making TWiR one of the community's favourite newsletters. We wish him all the best in his future projects.

Reviewer 2: Maybe use "Vikrant Chaudhary (@nasa42)"?
